api/python/provisioner/commands/setup_provisioner.py

```diff
@@ -483,6 +510,15 @@ def __attrs_post_init__(self):  # noqa: C901
                     "for bundle distribution type"
                 )
                 self.url_cortx_deps = None
+            if (
+                not self.disable_pip_repo and
+                self.dist_type != config.DistrType.BUNDLE
+            ):
```

Comment:
Should this if be modified to:
```
if (
        self.pypi_repo or
        self.dist_type == config.DistrType.BUNDLE
):
```

Reply:
@ypise NOT to both with AND operator does not work as expected, hence didn't change.
https://stackoverflow.com/questions/33565519/python-if-not-this-and-not-that/